3529/2/number_checker.rb

```diff
@@ -0,0 +1,44 @@
+require 'yaml'
+require 'redis'
+require 'redis-activesupport'
+require 'redis-rails'
+require 'redis-namespace'
+require 'redis-rack-cache'
+
+class NumberChecker
+  def initialize(file_path)
+    @file_path = file_path
+    @file = YAML.safe_load(File.read(@file_path))
+    @resp = ''
+  end
+
+  def handle_number(number, payload)
+    @resp = ''
+    @file['participents'].each do |participant|
+      @resp = checker_id(participant, payload, number) if number == participant.keys.first.to_s
```

Comment:
Clarify this one

Reply:
if users number exist in database of camp i need to check if there is signed in user with this number